Cargo.toml

```diff
@@ -1,11 +1,22 @@
 [package]
-name = "pconvert-rust"
+name = "pconvert_rust"
```

Comment:
we really need to rename this to use the underscore?

Reply:
Changed it to follow the [Rust naming conventions](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html).
Furthermore, the `package` can be named `pconvert-rust` but the `lib` can not. So, for coherence and in order to follow the naming conventions, I suggest naming both `pconvert_rust`